packages/primecountpy/meta.yaml

```diff
@@ -18,6 +18,9 @@ requirements:
     - libprimesieve
   run:
     - cysignals
+  constraint:
+    # ImportError: cysignals.signals does not export expected C function _do_raise_exception
+    - cysignals < 1.12.6
```

Comment:
This is https://github.com/passagemath/upstream-cysignals/issues/2 -- the people controlling cysignals broke the ABI in the middle of the release series